explanation/about-explanation.md

```diff
@@ -1,54 +1,48 @@
 
-Explanations (also called discussions), explain background information and provide context-specific knowledge on a particular topic.
+Explanation articles work well when they:
 
-The goals of an explanation article are understanding-oriented, not procedural or learning.
-
-Explanation articles work well when they deliver the following types of information:
-
-* They give context to a subject or topic that your audience may not have
-* They explain the "why" about a topic that may not be evident in reference or how-to articles
-* They suggest alternative approaches or different ways of thinking about a customer challenge or technical topic
-* They allow your audience to make connections between related concepts, so they can form a better understanding of the subject.
+* Give context to a subject or topic that your readers might not have
+* Explain the background about a topic that might not be evident in Reference or How To articles
+* Suggest alternative approaches or different ways of thinking about a use case or technical problem.
+* Help readers make connections between related concepts, and form a better understanding of the subject.
 
 ## Content of your explanation article
 
-### About the "Overview" section
-
-Summarise what the reader will get from reading the explanation article.
-
-Creating a good overview comes down to thinking about who is going to be using it:
+### The overview section
 
-* Are you writing only for end-users? For developers?
-* Are you writing only for people who have a certain problem to solve?
-* Is it intended for a particular industry.
+Summarize what the reader will achieve by reading the explanation article.
 
-### About the "Glossary" section
+* Are you writing for developers or for managers?
+* Are you writing for people who have a certain problem to solve?
+* Are you writing for a particular industry or market segment?
 
-If you plan on using a whole lot of terms that might not be familiar to your readers, list them all in this section.
+### The glossary section
 
+If you plan on using terms that might not be familiar to your readers, list them all in this section.
 Declaring them here lets you use any abbreviated forms of the terminology you're using in the body of the explanation.
 
 ### Structuring explanation articles
 
-Think about structuring explanation topics like you would a presentation to a group of folks who don't know anything about your chosen subject.
+Think about structuring explanation topics like you would a presentation to a group who don't know anything about your chosen subject.
 Your explanation should introduce ideas gradually, so your audience can grow their understanding of the concepts it covers.
 
-It can be easy to blend explanations with other types of content, like how-to or reference articles.
-Here are a few tips to avoid mixing documentation types:
+It can be easy to blend explanations with other types of content, like How To or Reference articles.
```

Comment:
I'm also on the fence about these template names as uppercase. 

"like how-to or reference articles". 

It's probably a wider discussion about how we treat our template names and is outside the scope of this PR.